content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Troubleshooting Guide
+description: This article helps common troubleshooting issues with XSOAR integrations
+---
+
+## Connection issues troubleshooting guide:
+### When a customer has any connection errors try the following steps:
+1. Check and uncheck the “insecure” checkbox
+2. Check and uncheck the “proxy” checkbox
+3. Try to isolate the problem, is it the integration instance server or the client?
+
+    a. In case you have access from your local computer to the integration instance server, Try to generate the same request using curl or postman from your local computer (or customer’s computer)- if it fails: the integration instance server is probably down, could be instance issue.
+    
+    b. Try to do some curl from the XSOAR machine, for example: 
+    ```
+   curl https://httpbin.org/status/200  -vk
+   ```
+    and see if the returning status is 200
+
+    c. Try to run some curl request via a docker container using 
+    ```
+   docker run -it demisto/netutils:1.0.0.6138 curl https://httpbin.org/status/200  -vk
+   ```
+   
+    - If section c. fails and you suspect this is a proxy issue, try to run curl without proxy using 
+    ```
+   docker run -it demisto/netutils:1.0.0.6138 curl https://httpbin.org/status/200  -vk --noproxy "*"
+   ```
+   
+    - If section b. worked and section c. failed: this could be a docker networking issue, see the following [link](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/5-5/cortex-xsoar-admin/docker/docker-hardening-guide/troubleshoot-docker-networking-issues) for guidance.
+    - If section a. worked but sections b. and c. failed: This could possibly be an integration bug, open an issue with server log bundle and preferably a .pcap files of the commands attempts.
+4. See if the customer’s firewall is blocking some packets during a command. Since some integrations has some internal redirection to different destination IPs. See this issue as an example
+
+### Diagram for section 3:
+![](https://raw.githubusercontent.com/demisto/content-docs/ec010688ce513b599bde8cdc8b5ad5c743526a8a/static/img/troubleshooting_diagram.jpg)
```

Comment:
the failed option is the same as the working option:
![image](https://user-images.githubusercontent.com/30797606/84357229-00028800-abce-11ea-96d8-f51a6b771870.png)

if intentional, no need to split the cases.